packages/web/src/common/store/upload/sagas.ts

```diff
@@ -1072,10 +1074,20 @@ export function* uploadMultipleTracks(
   })
 
   // Get true track metadatas back
-  const newTracks = yield* call(retrieveTracks, {
-    trackIds,
-    forceRetrieveFromSource: true
-  })
+  // Allow for retries in case the tracks are not immediately available
```

Comment:
Is this possible?

Reply:
could be? technically the confirmer is only waiting for block numbers

Reply:
If the indexer passed the block number that the uploads went in on, the track should be there... Also I think in most cases we see this error the tracks _didnt_ upload successfully